From 7becfd45c7dc03fdb618efcd9ec34cd9238d5732 Mon Sep 17 00:00:00 2001
From: Mohamed <Mohamed@gmail.com>
Date: Tue, 19 Dec 2023 01:14:02 +0100
Subject: [PATCH] Boot-Static-IP-HomeWifi

---
 interfaces | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/interfaces b/interfaces
index 0acf4cf..fb15701 100644
--- a/interfaces
+++ b/interfaces
@@ -5,7 +5,12 @@ auto lo
 iface lo inet loopback
 
 # Wireless interfaces
-iface wlan0 inet dhcp
+auto wlan0 
+iface wlan0 inet static
+	address 192.168.1.31
+	netmask 255.255.255.0
+	gateway 192.168.1.1
+	dns-nameserver 8.8.8.8 
 	wireless_mode managed
 	wireless_essid any
 	wpa-driver wext
@@ -14,7 +19,6 @@ iface wlan0 inet dhcp
 iface atml0 inet dhcp
 
 # Wired or wireless interfaces
-auto eth0
 iface eth0 inet dhcp
 iface eth1 inet dhcp
 
-- 
2.25.1

